### PR TITLE
bump go-fastly version

### PR DESCRIPTION
**Pull request by smaeda-ks**
Full test passed.

### CODE REVIEW

File: fastly/resource_fastly_service_acl_entries_v1.go [hunk 1]
```diff
@@ -74,6 +76,11 @@ func resourceServiceAclEntriesV1() *schema.Resource {
 	}
 }
 
+func convertSubnetToInt(s string) int {
```
Comment (bengesoff) [inline fastly/resource_fastly_service_acl_entries_v1.go]: I think normally helper functions such as these go at the bottom of the file

**Review by bengesoff — APPROVED** [review]
Looks good, one minor nitpicky comment but otherwise 👍🏼

File: fastly/resource_fastly_service_acl_entries_v1.go (hunk 1 above)
Comment (smaeda-ks) [reply fastly/resource_fastly_service_acl_entries_v1.go]: Thanks! Moved that to the bottom

File: fastly/block_fastly_service_v1_dynamicsnippet.go [hunk 2]
```diff
@@ -157,30 +154,7 @@ func buildDynamicSnippet(dynamicSnippetMap interface{}) (*gofastly.CreateSnippet
...
-	case "error":
-		opts.Type = gofastly.SnippetTypeError
-	case "deliver":
-		opts.Type = gofastly.SnippetTypeDeliver
-	case "log":
-		opts.Type = gofastly.SnippetTypeLog
-	case "none":
-		opts.Type = gofastly.SnippetTypeNone
-	}
+	opts.Type = gofastly.SnippetType(snippetType)
```
Comment (Integralist) [inline fastly/block_fastly_service_v1_dynamicsnippet.go:157]: Nice! 🙂

**Review by Integralist — APPROVED** [review]
LGTM

**Pull request merged**